### DISCUSSION TITLE
Deleting account

### DISCUSSION THREAD

**Discussion by insooeric** [planetscale/discussion#551] [Q&A] [answered]
Hi,
I accidentally made 2 databases which I never used and ended up paying around $18. I tried to contact Planetscale via email, but I haven't gotten any response yet.
So, i removed all databases and decided to delete the account to prevent further payments, but it's saying "Please pay any pending invoices and try again."
Is there any way I can cancel the invoice?

EDIT: I just figured out they charged an additional $29.

**Comment by orware** [comment, marked as answer]
Hello @insooeric,

Your request sounded familiar and I believe your issues were all addressed via the ticket that you also submitted yesterday where my colleague was able to assist you. 

If anything else is needed though, feel free to send in a new reply to your ticket and we'll be happy to help!

**Reply by sahidAleDev** [reply]
the same thing happened to me, how can I remove my account?

**Comment by sahidAleDev** [comment]
the same thing happened to me, how can I remove my account?

**Reply by mscoutermarsh** [reply]
Hi @sahidAleDev. Please email support: https://support.planetscale.com/hc/en-us. It's hard for us to solve problems like this on the public discussion board.
